Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -80,14 +80,23 @@ open class _ChatMessageActionsVC<ExtraData: ExtraDataTypes>: _ViewController, Th
             message.deletedAt == nil
         else { return [] }
 
+        let channelConfig = messageController.client
+            .channelController(for: messageController.cid)
+            .channel?
+            .config
+
         switch message.localState {
         case nil:
             var actions: [ChatMessageActionItem] = [
-                inlineReplyActionItem(),
-                threadReplyActionItem(),
-                copyActionItem()
+                inlineReplyActionItem()
             ]
 
+            if channelConfig?.repliesEnabled == true {
+                actions.append(threadReplyActionItem())
+            }
```

Comment:
Shouldn't we also disable inline replies if `repliesEnabled == false`? 🤔 The docs on `repliesEnabled` say: `Enables message threads and replies. Enabled by default.`

Reply:
I've updated the documentation